### PR TITLE
Run app in minikube on travis

### PR DESCRIPTION
**Pull request by raboof**
Refs #8

### CODE REVIEW

File: scripts/setup-rp.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -exu
+
+# See https://developer.lightbend.com/docs/lightbend-orchestration/current/setup/cli-installation.html#install-the-cli for instructions for your platform
+
+wget -qO - https://downloads.lightbend.com/rp/keys/bintray-debian | \
+    sudo apt-key add - && \
```
Comment (chbatey) [inline scripts/setup-rp.sh]: should we maybe stop using reactive-app? Another PR tho

Comment (raboof) [reply scripts/setup-rp.sh]: Yeah I don't think it adds much in this case - I didn't realize (until I prepared this PR) that `sbt 'deploy minikube'` actually needs the rp tool to be installed. Added #12

**Comment by raboof** [comment]
(it is now failing on https://github.com/lightbend/sbt-reactive-app/blob/master/src/main/scala/com/lightbend/rp/sbtreactiveapp/Deploy.scala#L106, added that command to the script so we can diagnose)

**Comment by raboof** [comment]
This is a known issue tracked as https://github.com/lightbend/sbt-reactive-app/issues/107

**Pull request merged**